tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -51,7 +51,7 @@ def stop_producer_and_consumer(self):
 
         # Check that producer is still successfully producing
         currently_acked = self.producer.num_acked
-        wait_until(lambda: self.producer.num_acked > currently_acked + 5, timeout_sec=10,
+        wait_until(lambda: self.producer.num_acked > currently_acked + 5, timeout_sec=360,
```

Comment:
Not sure if it's related, but there was a bug in `SslTransportLayer` that was fixed yesterday that would cause the producer to time out sometimes while waiting for metadata response:

https://issues.apache.org/jira/browse/KAFKA-2801